static/.eslintrc

```diff
@@ -41,18 +41,39 @@ Do not edit this file in your local project
     "comma-dangle": [2, "never"],
     "comma-spacing": [2, { "before": false, "after": true }],
     "comma-style": [2, "last"],
+    "computed-property-spacing": [2, "never"],
     "constructor-super": 2,
     "curly": [2, "multi-line"],
     "dot-location": [2, "property"],
-    "eol-last": 2,
+    "eol-last": [2, "always"],
     "eqeqeq": [2, "allow-null"],
     "func-call-spacing": [2, "never"],
     "handle-callback-err": [2, "^(err|error)$" ],
-    "indent": [2, 2, { "SwitchCase": 1 }],
+    "indent": [2, 2, {
+      "SwitchCase": 1,
+      "VariableDeclarator": 1,
+      "outerIIFEBody": 1,
+      "FunctionDeclaration": {
+        "parameters": 1,
+        "body": 1
+      },
+      "FunctionExpression": {
+        "parameters": 1,
+        "body": 1
+      }
+    }],
     "key-spacing": [2, { "beforeColon": false, "afterColon": true }],
     "keyword-spacing": [2, { "before": true, "after": true }],
+    "linebreak-style": [2, "unix"],
+    "max-len": ["error", {
+      "code": 100,
+      "ignoreRegExpLiterals": true,
+      "ignorePattern": "^\\s*const\\s.+=\\s*require\\s*\\(/"
+    }],
+    "multiline-ternary": ["off", "never"],
```

Comment:
Hmm. they are more readable than on one line.
IMO prevent nested ternaries is good enough.